NAMESPACE

```diff
@@ -1,5 +1,6 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method("[",qenv)
```

Comment:
Any way to have this method be documented as the last one in the documentation, not the first one?
<img width="493" alt="image" src="https://github.com/user-attachments/assets/7916ba55-353a-4c99-8d21-e8ddfd0a2720">

Reply:
please see `within.qenv`. `within` is listed in [package index](https://insightsengineering.github.io/teal.code/latest-tag/reference/qenv.html)

Reply:
I think I didn't rebuild the documentation locally. This is how it looks, and it's proper now

The order is taken by the alphabetical order of R-files

- qenv -> qenv-constructor.R
- new_qenv -> qenv-constructor.R
- eval_code -> qenv-eval_code.R
- x[datanames] -> qenv-extract.R
- get_code -> qenv-get_codeR
- within -> qenv-withinr.R

<img width="481" alt="image" src="https://github.com/user-attachments/assets/46fc35cf-7e56-4698-94f7-e0f1951adfb5">